content/docs/2019-06-23-introducing-mlrplayground.Rmd

```diff
@@ -0,0 +1,120 @@
+---
+title: "Introducing mlrPlayground"
+authors: ["sebastian-gruber"]
+date: "2019-06-23"
+output:
+  blogdown::html_page:
+    toc: true
+categories:
+  - r-bloggers
+tags:
+  - shiny
+  - visualization
+  - mlr
+  - teaching
+  - classification
+  - regression
+  - hyperparameter
+---
+
+<img src="https://i.imgur.com/MPEwCJ2.gif" width=800 />
+<br/>
+
+## First of all
+You may ask yourself how is this name ‘mlrPlayground’ even justified?
+What a madman puts two such opposite terms in a single word and expects people to take him seriously?
```

Comment:
I like catchy starts but this one is maybe a bit over the top :) At least "madman" should be replaced.

Reply:
done